bucket/macast.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.7",
+    "description": "A menu bar application using mpv as DLNA Media Renderer. You can push videos, pictures or musics from your mobile phone to your computer.",
+    "homepage": "https://github.com/xfangfang/Macast",
+    "license": "GNU General Public License v3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/xfangfang/Macast/releases/download/v0.7/Macast-Windows-v0.7.exe",
+            "hash": "sha1:6400b23f29232af36f20ae1586ad28b05e5a79e9"
+        }
+    },
+    "bin": [["Macast-Windows-v0.7.exe", "macast" ]],
```

Comment:
Commandline arguments?

Reply:
@Lutra-Fs what about this?

Reply:
if we do like the following changes then line 12 should be changed as 
``` "bin":"macast.exe", ```

> > Commandline arguments?
> 
> @Lutra-Fs what about this?

Sry, I am a little bit confused what you mean.

Reply:
the software is not CLI software.

Reply:
> the software is not CLI software.

Then we can just remove this line